server/application-server/src/main/java/de/tum/in/www1/hephaestus/codereview/pullrequest/PullRequest.java

```diff
@@ -36,6 +30,9 @@ public class PullRequest extends BaseGitServiceEntity {
     @NonNull
     private String url;
 
+    @NonNull
+    private int prNumber;
```

Comment:
I already included the `number`-variable this in another PR . When you bring this branch up-to-date with `develop` you can just use that one instead.